robolectric-shadows/shadows-core/pom.xml

```diff
@@ -389,9 +427,29 @@
       </build>
       <dependencies>
         <dependency>
+          <groupId>com.android.tools.layoutlib</groupId>
```

Comment:
On maven they only support 21+, I tried to build with android source, and they are not part of the Android framework, the source of those tools sits in https://android.googlesource.com/platform/tools/base/, looks like not something the script is handling now. Again we can worry about compatibility later.